libhdf5/hdf5var.c

```diff
@@ -22,6 +22,9 @@
  * order. */
 #define NC_TEMP_NAME "_netcdf4_temporary_variable_name_for_rename"
 
+/** Number of bytes in 64 MB. */
+#define SIXTY_FOUR_MB (67108864)
+
```

Comment:
The limit for a compact data set is 64 KiB, not 64 MiB

Reply:
Thanks, I will fix.